study-builder/studies/shared/family-history/family-history-about-you.conf

```diff
@@ -0,0 +1,271 @@
+{
+  include required("../../snippets/activity-general-form.conf"),
+  "studyGuid": ${id.study},
+  "activityCode": "FAMILY_HISTORY_ABOUT_YOU",
+  "parentActivityCode": "FAMILY_HISTORY",
+  "createOnParentCreation": true,
+  "maxInstancesPerUser": 1,
+  "displayOrder": 5,
+  "versionTag": "v1",
+  "allowUnauthenticated": false,
+  "listStyleHint": "NONE",
+
+  OPTION_ATM={
+    include required("snippets/picklist-option.conf"),
+    LABEL_EN="Ataxia telangiectasia [ATM]"
+    stableId="ATM"
+  },
+  OPTION_CONSTITUTIONAL_MISMATCH_REPAIR_DEFI_SYNDROME={
```

Comment:
These options are defined here because our implementation of the HOCON parser will not do substitutions on includes from within arrays!
The options are used below.